tests/data_models/inspector/test_fixed_combination.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import pytest
+
+from sdgx.data_models.inspectors.fixed_combination import FixedCombinationInspector
+
+
+@pytest.fixture
+def test_fixed_combination_data():
```

Comment:
Here we need assert two case for fixed combination.
```suggestion
@pytest.fixture
def test_fixed_combination_data():
    data = {
        "A": [1, 2, 3, 4, 5],
        "B": [2, 4, 6, 8, 10],  # B is 2 * A
        "C": [5, 5, 5, 5, 5],  # C is constant
        "D": [1, 3, 5, 7, 9],  # D is not related to A or B
        "E": [2, 4, 6, 8, 10],  # E is 2 * A
        "categorical_one": ["co1", "co3", "co2", "co9", "co1"], # categorical_one and categorical_two have a one-to-one correspondence.
        "categorical_two": ["ct1", "ct3", "ct2", "ct9", "ct1"]
    }
    df = pd.DataFrame(data)
    yield df
```